astropy/table/table.py

```diff
@@ -150,6 +150,19 @@ def keys(self):
     def values(self):
         return list(OrderedDict.values(self))
 
+    def isinstance(self, *cls):
+        '''
```

Comment:
Use `"""` instead of `'''` to follow the existing style of the `table.py` module.